backend/tests/src/tests/specs/users/users-delete/users-delete.spec.ts

```diff
@@ -0,0 +1,156 @@
+    apiSessionStorage.addAndEnterSession('user to delete');
+    expectedSignUpResponse = {
+      email: userToDeleteData.email,
+    };
+
+    expectedUserDetails = {
+      fullName: userToDeleteData.fullName,
+    };
+
+    const response = (await authService.signUp(
+      userToDeleteData,
+    )) as Response<UserSignUpResponseDto>;
+
+    httpService.setToken(response.body.token);
+    userToDeleteId = response.body.user.id;
+
+    apiSessionStorage.exitSession();
+
+    response.should.have.status(HttpCode.CREATED);
+    response.should.have.normalExecutionTime;
+    response.body.should.have.jsonSchema(signUpResponseSchema);
+    response.body.user.should.deep.include(expectedSignUpResponse);
+    response.body.user.userDetails.should.deep.include(expectedUserDetails);
+    JWT_TOKEN_REGEX.test(response.body.token).should.be.true;
+  });
+
+  before('should log in as a UAM manager', async () => {
+    apiSessionStorage.addAndEnterSession('UAM Manager');
+    const response = (await authService.signIn(
+      testsConfig.users.uamManager,
+    )) as Response<UserSignInResponseDto>;
+    httpService.setToken(response.body.token);
+    apiSessionStorage.exitSession();
+
+    response.should.have.status(HttpCode.OK);
+    response.should.have.normalExecutionTime;
+    response.body.should.have.jsonSchema(signInResponseSchema);
+    JWT_TOKEN_REGEX.test(response.body.token).should.be.true;
+  });
+
+  before('should log in as a student', async () => {
+    apiSessionStorage.addAndEnterSession('student');
+    const response = (await authService.signIn(
+      testsConfig.users.student,
+    )) as Response<UserSignInResponseDto>;
+    httpService.setToken(response.body.token);
+    apiSessionStorage.exitSession();
+
+    response.should.have.status(HttpCode.OK);
+    response.should.have.normalExecutionTime;
+    response.body.should.have.jsonSchema(signInResponseSchema);
+  });
+
+  it(`should return an error after deleting user by user without Manage UAM permission:
+        email:${userToDeleteData.email}
+        password:${userToDeleteData.password}`, async () => {
+    apiSessionStorage.enterSession('student');
+    const response = (await usersService.delete(
+      userToDeleteId,
+    )) as Response<HttpErrorDto>;
+    apiSessionStorage.exitSession();
+
+    response.should.have.status(HttpCode.FORBIDDEN);
+    response.should.have.normalExecutionTime;
+    response.body.should.have.jsonSchema(errorResponseSchema);
+    response.body.error.should.be.equal(HttpStatusMessage.FORBIDDEN);
+    response.body.message.should.be.equal(ExceptionMessage.PERMISSION_LACK);
+  });
+
+  it('should delete user by user with Manage UAM permission', async () => {
+    apiSessionStorage.enterSession('UAM Manager');
+    const response = await usersService.delete(userToDeleteId);
+    apiSessionStorage.exitSession();
+
+    response.should.have.status(HttpCode.OK);
+    response.should.have.normalExecutionTime;
```

Comment:
```suggestion
    response.should.have.status(HttpCode.OK);
    response.should.have.normalExecutionTime;
    response.body.should.be.equal(true);
```

And add `response.body.should.be.equal(true);` everywhere where user delete request should succeed